### DISCUSSION TITLE
Default Deck

### DISCUSSION THREAD

**Discussion by EvaTheDM** [EvaTheDM/mildly-magical-inspiration#6] [Q&A]
On the Roadmap for the future of this module is a default deck as content to come with the module. I‘d be happy to hear about your suggestion for cards to fill up such a default deck, both when it comes to card titles as well as card effect.

**Comment by cgplayer2000** [comment]
The most obvious suggestion would be a poker deck of 52 cards, 4 suits, 13 cards per suit (Ace to King).  If you included the csv or json for your default deck, then it'd also serve as a tutorial.

**Reply by EvaTheDM** [reply]
The module wasn't really intended to be used as an actual playing card deck, as such for each card it is asking for a title, subtitle, activation cost, duration, etc. and is also missing any functionality to use it to play a card game. I'll put adding a pocker deck as one default option onto my list, but I feel that it wouldn't be an ideal example of how the module was designed to be used. But still, thanks for the suggestion!

**Reply by cgplayer2000** [reply]
Ah, that makes sense.  But, the csv/json of the deck would serve as a really useful tutorial for how to set up our own decks.  Thanks for responding.
